stack.yaml

```diff
@@ -61,6 +61,8 @@ extra-deps:
 # Need https://github.com/phadej/github/pull/387 and to be added back in to stackage
 - git: https://github.com/shmish111/github.git
   commit: cc27b9de4d5d0939235fa9a8b418de3ea4807bab
+- git: https://github.com/input-output-hk/iots-export.git
```

Comment:
Oh lol I was waiting to find where this actually was! To be clear, when I said it could be a separate library, I just meant another package in this repo! However, maybe this is actually helpful for other people consuming our output?

Reply:
I'm also not really sure how to leave review comments on a whole other repository...

Reply:
Oh, I see! I thought you meant a separate package. A suggestion I thought was a very good one. It really doesn't have anything to do with plutus _per-se_.

I can roll it back in if you like. My vote is to keep it standalone _but_ get it into CI.

Reply:
Well, our repository does contain many packages... perhaps we had a terminology clash there :p 

I forsee the library being pretty tightly coupled to our producers and consumers *at the moment*. Maybe one day we should split it out, but for the moment I think it's going to be more trouble than it's worth.

Reply:
Fair enough. I'll pull it back.